src/api/app/controllers/webui/package_controller.rb

```diff
@@ -386,9 +386,6 @@ def set_file_details
       @spec_count += 1 if file[:ext] == 'spec'
     end
 
-    # check source service state
-    @package.serviceinfo.value(:error) if @package.serviceinfo
```

Comment:
did I oversee any side effect on this one?

Reply:
I don't think so. This 'just' fetched the file.

Reply:
Returning true since 2012 :)
16be2f5dcc21c608bf5221baba3259357690c940

Reply:
Returning true to void since 2012